modules/ChEMBL.py

```diff
@@ -30,7 +30,7 @@ def main(chembl_evidence: str, predictions: str, output_file: str) -> None:
     logging.info(f'Classes of reason to stop table: {predictions}')
 
     # Load input into dataframes
-    chembl_df = spark.read.json(chembl_evidence).persist()
+    chembl_df = spark.read.json(chembl_evidence).repartition(20).persist()
```

Comment:
Why do you need repartitioning the data?

Reply:
When reading data from JSON, Spark loads everything in the main node rather than splitting it across multiple workers. Since ChEMBL evidence is >300k records, I thought it would be beneficial to redistribute that chunk of data in different nodes to make the join more performant.

That was the theory. Reality is this:
```python3
In [34]: timeit -n 3 spark.read.json(chembl).join(predictions_df, on='studyStopReason', how='left').distinct()
2.48 s ± 8.3 ms per loop (mean ± std. dev. of 7 runs, 3 loops each)

In [35]: timeit -n 3 spark.read.json(chembl).repartition(20).join(predictions_df, on='studyStopReason', how='left').distinct()
2.48 s ± 12.1 ms per loop (mean ± std. dev. of 7 runs, 3 loops each)
```

No actual benefit when repartitioning. Thanks for flagging, Ill remove the statement.